tests/test_methods_nai.py

```diff
@@ -0,0 +1,58 @@
+"""This test aims to compare to previous PDA and PDAW results that were published and compared to exact QD.
+We use NaI as an example with two different laser pulses. If this test fails, the results produced by the code are unreliable!"""
+
+import numpy as np
+
+from promdens.promdens import InitialConditions, LaserPulse
+
+
+def test_pdaw_nai(make_pulse):
+    ics = InitialConditions(nsamples=1000, nstates=1, input_type='file')
+
+    # todo: this file path works only if 'pytest' is launched in the main folder (the same for all later)
+    ics.read_input_data(fname='tests/NaI_reference/test_data_nai.dat', energy_unit='a.u.', tdm_unit='a.u.')
+
+    fstoau = 41.341374575751
+    pulse = LaserPulse(omega=0.13520905, fwhm=20*fstoau, envelope_type='gauss', lchirp=0, t0=0)
+
+    ics.calc_field(pulse=pulse)
+
+    ics.windowing()
+
+    # the following command was used to generate the reference
+    # promdens -m pdaw -n 1000 -w 0.13520905 -f 20 test_data_nai.dat
+    # the reference was compared to exact QD
+    reference = np.genfromtxt('tests/NaI_reference/test_pdaw_reference.dat').T[1]
+    weights = ics.weights[0]
+
+    assert len(weights) == len(weights)
+    # comparing all weights
+    for i in range(len(weights)):
+        assert weights[i] == reference[i]
```

Comment:
This will make the output of a failed test little more informative
```suggestion
        assert weights[i] == reference[i], f"pdaw weight[{i}] does not match: delta={weights[i]-reference[i]}"
```